lua/wikis/commons/Class.lua

```diff
@@ -110,40 +104,14 @@ function Class._wrapFunction(f, options)
 			)
 
 		if shouldRewriteArgs then
```

Comment:
Should the rewrite args handler be removed? Seems like it will just produce an unexpected results sometimes.
Or is that on your todo?
(OOS)

Reply:
isn't it still needed to determine if Arguments.getArgs should be called or not?

Reply:
I can't think of a reason to not always call getArgs?

Reply:
you mean if frame is specified?
or also if it isn't frame?
(for the later case see comments with salt, same applies here)

Reply:
leaving it OOS for now, added it to #5818 as todo

Reply:
as a side note: it would break all modules that need #5828 (incl ones not tracked on this repo)